src/main/java/mate/academy/internetshop/controller/order/CompleteOrderController.java

```diff
@@ -27,6 +27,6 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         ShoppingCart shoppingCart = shoppingCartService.getByUserId(userId);
         List<Product> products = shoppingCartService.getAllProducts(shoppingCart);
         orderService.completeOrder(products, shoppingCart.getUser());
```

Comment:
```suggestion
        orderService.completeOrder(shoppingCart.getProducts(), shoppingCart.getUser());
```
щоб не звертатись до сервісу двічі

Reply:
+